backend/masterScript_createMapLayers.py

```diff
@@ -0,0 +1,78 @@
+# Import data sources information
+from data.dataSources import *
+
+# Import  functions to call
+from assimilator import assimilate
+from generateLayer import generateLayer
```

Comment:
`generateLayer.py` --> `generate_layer.py`

also, `generateLayer` is a function, therefore should be named `generate_layer` as well --> you would have spotted that (**automatically**) if you would have integrated the latest `pre-commit` integrations.

Therefore, this line will be: 

```
from generate_layer import generate_layer
```